lab_SOPs/onboardingstudents_SOP.md

```diff
@@ -0,0 +1,36 @@
+# Onboarding new students
+
+## Sharing access
+
+All students should have access to the following:
+1. `VKC LAB` folder on Google Drive
+2. `Klepac-Ceraj Group` Google calendar
+3. `VKC-Lab` on Slack
+    * All students should be added to `#general` channel to start
+    * Other generally relevant channels include `#papers`, `#random`, and `#resources`
+
+## Physical access
+
+All students should have swipe access to L-wing. If this is not the case, Vanja can share their student B number with *insert admin name* and ask that the student be granted after-hour access.
+
+All students will be given a designated third of a shelf in L307. These can be designated with a piece of colored lab tape with their name.
+
+All students who will be doing any wet-lab work should also be given a new [lab notebook](https://www.amazon.com/stores/node/14640355011?_encoding=UTF8&field-lbr_brands_browse-bin=Vela%20Sciences&ref_=bl_dp_s_web_14640355011) (*double check brand of notebook*).
+
+## Individual documents and folders on Google Drive
+
+Students should create their own folder in `VKC LAB > Personal & Project Folders > FIRSTNAME LASTNAME 'YY` ('YY = class year) for storing any miscellaneous documents related to their research.
+
+Students should create their own running agenda for their weekly meetings in `VKC LAB > Individual Weekly Meetings`. This can be done by making a duplicate of `WeeklyAgenda_TEMPLATE` and renaming it `WeeklyAgenda_YOURNAME`.
+
+## Member information for Airtable and [the lab website](https://www.vkclab.com/)
+
+All students should fill out the [new member form](https://airtable.com/shr3kbs51ADf4ztJA) within a week of joining the lab.
+
+*Note: all lab members should be reminded at the beginning of every semester to fill out the form if any information requires updating.*
+
+This form will populate [this list](https://airtable.com/shrxfBjuMkFFTd1E8) on Airtable.
+
+*Note: if a members answers aren't appearing, it may be because a staff member has not designated `Yes` in the `Use this version?` column which allows older, out-of-date submissions to be filtered out.
+
+**FOR STAFF**: The answers to this form should be used to update the [lab website](https://www.vkclab.com/our-team) no later than *one month* within the member joining.
```

Comment:
Can you add section:

````
## Access to `ada`
### For new students

```sh
adduser --ingroup students $USER
```

Get public key,
and add to `/home/$USER/.ssh/authorized_keys`

### For new staff

```sh
adduser --ingroup facstaff $USER
usermod -aG students $USER
usermod -aG sudo $USER # optional
```

Get public key,
and add to `/home/$USER/.ssh/authorized_keys`
````